src/main/java/github/GitHubEventsResponse.java

```diff
@@ -42,7 +45,14 @@ private void parseEventParameters(InputStream jsonBody) {
         BufferedReader reader;
         try {
             reader = new BufferedReader(new InputStreamReader(jsonBody, CHARSET_UTF8));
-            Map<String, Object>[] eventsWithParameters = gson.fromJson(reader, type);
+            Optional<Map<String, Object>[]> possibleParameters =
+                Optional.ofNullable(gson.fromJson(reader, type));
+
+            if (!possibleParameters.isPresent()) {
+                return;
+            }
+            Map<String, Object>[] eventsWithParameters = possibleParameters.get();
+
```

Comment:
Hmm, actually no, that's overkill :stuck_out_tongue:  I thought I'd be propagating the empty object but it didn't turn out to be needed.